docs/pages/linking/into-your-app.mdx

```diff
@@ -7,7 +7,7 @@ description: Learn how to handle an incoming URL in your React Native and Expo a
 import { Collapsible } from '~/ui/components/Collapsible';
 import { Terminal } from '~/ui/components/Snippet';
 
-This guide provides steps to configure standard **deep links** in your project by adding a custom scheme.
+This guide provides steps to configure standard **deep links** in your project by adding a custom scheme. For production apps, you probably want to (also) set up app [Universal linking](/linking/overview/#universal-linking).
```

Comment:
Techbnically, Universal Links is the iOS term and the equivalent concept on Android is called App Links. The Android [docs](https://developer.android.com/training/app-links) on the topic don't use the term "universal" at all.

```suggestion
This guide provides steps to configure standard **deep links** in your project by adding a custom scheme. For production apps, you probably want to (also) set up app [Universal / App Links](/linking/overview/#universal-linking).
```